ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -150,97 +145,105 @@ function NetworkGraphPage() {
             clusterFromUrl && namespacesFromUrl.length > 0 && deploymentCount;
 
         if (isQueryFilterComplete && selectedClusterId && isClusterNamespaceSelected) {
-            if (nodeUpdatesCount === 0) {
-                setIsLoading(true);
+            setIsLoading(true);
 
-                const queryToUse = queryService.objectToWhereClause(remainingQuery);
-                const timestampToUse = timeWindowToDate(timeWindow);
+            const queryToUse = queryService.objectToWhereClause(remainingQuery);
+            const timestampToUse = timeWindowToDate(timeWindow);
 
-                Promise.all([
-                    // fetch the network graph data used for the active graph
-                    fetchNetworkFlowGraph(
-                        selectedClusterId,
-                        namespacesFromUrl,
-                        deploymentsFromUrl,
-                        queryToUse,
-                        timestampToUse || undefined,
-                        includePorts,
-                        ALWAYS_SHOW_ORCHESTRATOR_COMPONENTS
-                    ),
-                    // fetch the network graph data, including policies, for the inactive graph
-                    fetchNetworkFlowGraph(
-                        selectedClusterId,
-                        namespacesFromUrl,
-                        deploymentsFromUrl,
-                        queryToUse,
-                        undefined,
-                        includePorts,
-                        ALWAYS_SHOW_ORCHESTRATOR_COMPONENTS,
-                        INCLUDE_POLICIES
-                    ),
-                ])
-                    .then((values) => {
-                        // get policy nodes, and the starting epoch, from policy graph API response
-                        const { nodes: policyNodes, epoch } = values[1].response;
```

Comment:
This `epoch` value is _always_ zero. It seems like this should match the value from the epoch-specific endpoint so we know what iteration of the graph data is being returned, which would allow us to avoid an extra request when the graph first loads.

Is this a BE bug, or an API change?

Reply:
^ cc: @charmik-redhat @md2119

Reply:
What is the exact API call?

Reply:
Any call to `/v1/networkgraph/cluster/<clusterId>` returns ` epoch: 0` as part of the response.

e.g. Updating the graph parameters on staging: https://stagingdb.demo.stackrox.com/v1/networkgraph/cluster/016d4fd9-88fc-4752-b7b5-9272a79009f5?query=Namespace%3Afrontend%2Cbackend&since=2023-06-22T18%3A41%3A38.046Z&includePorts=true

Reply:
The epoch in the request is never set. The API can be updated to add the correct epoch. You should reach out to Maple team.

Reply:
Ok I'll make a note to coordinate with them. Thanks for taking a look!